wgpu-types/src/counters.rs

```diff
@@ -101,31 +101,31 @@ impl std::fmt::Debug for InternalCounter {
 #[derive(Clone, Default)]
 pub struct HalCounters {
     // API objects
-    ///
+    #[allow(missing_docs)]
```

Comment:
We can just stick one of these on the struct and not need to duplicate it everywhere

Reply:
butbutbut what if we forget the others that _need_ docs 😱

Guess this is reasonable, if not my preference. Fixed with [my latest push](https://github.com/gfx-rs/wgpu/compare/d652aa5eca6b80bdfe69d35128b1fea8cfdeed77..8ace14736f33002cf9e630118d9300c669c9b27f).